ci.sh

```diff
@@ -2,28 +2,17 @@
 
```

Comment:
Instead of the `|| exit`s everywhere, per a tip [from MIT's CS club](https://sipb.mit.edu/doc/safe-shell/), you can put

```
set -euf -o pipefail
```

at the top of this script.

I am also 99% sure that GH Actions has the equivalent set; I've seen it stop builds early lots of times when any of the intermediate steps fail.

Reply:
Good call, thanks! Resolved in ed38c19 and bd78ac6